app/models/account.rb

```diff
@@ -0,0 +1,22 @@
+class Account < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
+  
+  has_many :posts
+  has_many :likes
+  has_many :comments       
+  protected
+    def password_required?
+      confirmed? ? super : false
+    end
```

Comment:
Extra indentation the methods are parallel to private, protected block.

```suggestion
  protected
  def password_required?
    confirmed? ? super : false
  end
```

Reply:
okay